Zend/zend_string.h

```diff
@@ -442,6 +442,7 @@ EMPTY_SWITCH_DEFAULT_CASE()
 	_(ZEND_STR_NAME,                   "name") \
 	_(ZEND_STR_ARGV,                   "argv") \
 	_(ZEND_STR_ARGC,                   "argc") \
+	_(ZEND_STR_ARRAY_CAPITALIZED,      "Array") \
```

Comment:
I get _why_ it's called `ZEND_STR_ARRAY_CAPITALIZED`, _but_ I found it confusing in the places where actually used. Would something like `ZEND_STR_ARRAY_TYPE_STRING` be more descriptive?

Reply:
For `ARRAY_TYPE_STRING` I would expect this to be the lower-case variant `"array"`, as that's how it's written as a type.